src/Socket_p.h

```diff
@@ -352,14 +364,14 @@ namespace Arcus
         uint32_t header = (ARCUS_SIGNATURE << 16) | (VERSION_MAJOR << 8) | (VERSION_MINOR);
         if(platform_socket.writeUInt32(header) == -1)
         {
-            error(ErrorCode::SendFailedError, "Could not send message header");
+            error(ErrorCode::SendFailedError, "Could not send message header 555");
```

Comment:
Please remove the debug code here.
```suggestion
            error(ErrorCode::SendFailedError, "Could not send message header");
```